Verification/FLiBe_Prop_Verification.py

```diff
@@ -0,0 +1,164 @@
+df6.plot(x='Temp', y='Density', label='Janz 1974/1988', ax=ax, linestyle='--', color='darkorange')
+df7.plot(x='Temp', y='Density', label='Recommended', ax=ax, linestyle='-.', color='cornflowerblue')
+df8.plot(x='Temp', y='Density', label='Zaghoul 2003', ax=ax, linestyle='--', color='lime')
+plt.plot(T,Rho,'r--', label='Density Used', linewidth=3, dashes=(10,20))
+plt.xlabel('Temperature - K')
+plt.ylabel('Density - kg/m^3')
+plt.legend(loc='upper right')
+plt.grid()
+plt.title('Flibe Density')
+plt.show()
+
+data9 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Abe_1981_67.csv'))
+df9 = pd.DataFrame(data9, columns=['Temp','Vis'])
+data10 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Blanke_1956_62.csv'))
+df10 = pd.DataFrame(data10, columns=['Temp','Vis'])
+data11 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Blanke_1956_69-31.csv'))
+df11 = pd.DataFrame(data11, columns=['Temp','Vis'])
+data12 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Cantor_1969_64-36.csv'))
+df12 = pd.DataFrame(data12, columns=['Temp','Vis'])
+data13 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Cohen_1956_69-31.csv'))
+df13 = pd.DataFrame(data13, columns=['Temp','Vis'])
+data14 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Gierszewski_1980_66-34.csv'))
+df14 = pd.DataFrame(data14, columns=['Temp','Vis'])
+data15 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Janz_1974_64-36.csv'))
+df15 = pd.DataFrame(data15, columns=['Temp','Vis'])
+data16 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Cantor_1968_66-34.csv'))
+df16 = pd.DataFrame(data16, columns=['Temp','Vis'])
+
+
+fig2 = plt.figure(1, figsize=(10,8))
+ax = fig2.add_subplot()
+df9.plot(x='Temp', y='Vis', label='Abe 198 (67.2-32.8)', ax=ax, marker='^', linestyle='--', color='purple')
+df10.plot(x='Temp', y='Vis', label='Blanke 1956 (62.67-37.33)', ax=ax, marker='s', kind='scatter', color='dodgerblue')
+df11.plot(x='Temp', y='Vis', label='Blanke 1956 (69-31)', ax=ax, marker='x', kind='scatter', color='salmon')
+df12.plot(x='Temp', y='Vis', label='Cantor 1969 (64-36)', ax=ax, linestyle='-.', color='slategray')
+df13.plot(x='Temp', y='Vis', label='Cohen 1956 (69-31)', ax=ax, marker='o', linestyle='--', color='yellowgreen')
+df14.plot(x='Temp', y='Vis', label='Gierszewski 1980 (66-34)', ax=ax, linestyle=':', color='blue')
+df15.plot(x='Temp', y='Vis', label='Janz 1974 (64-36)', ax=ax, marker='+', kind='scatter', color='darkorange')
+df16.plot(x='Temp', y='Vis', label='Cantor 1968 (66-34)', ax=ax, linestyle='-', color='cyan')
+plt.plot(T2,Nu,'r--', label='Viscosity Used', linewidth=3, dashes=(10,20))
+plt.xlabel('Temperature - K')
+plt.ylabel('Viscosity - N/m^2/s')
+plt.title('Flibe Viscosity')
+plt.legend(loc='upper right')
+plt.grid()
+plt.show()
```

Comment:
Remove this plt.show(), only put one at the end so all of the plots show up at once.